### PR TITLE
Updated `$permissions` rules

### PR DESCRIPTION
**Pull request by Alex-302**
- added `Suppressing Google's "Protected" Audience API` https://github.com/AdguardTeam/Scriptlets/issues/395
- added `Suppressing Private State Token API` https://github.com/AdguardTeam/AdguardFilters/issues/175725
- changed `Suppressing both Google's Topics API and Topics API using Permissions policy` (related to `TODO: remove $document modifier after versions with Corelibs 1.14.38+ will be released`)

### CODE REVIEW

File: SpywareFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -403,17 +403,25 @@ accountkiller.com#%#window.ga = function(){var a = arguments[5];a&&a.hitCallback
...
 ! Suppressing both Google's Topics API and Topics API using Permissions policy
-! Works in CoreLibs-based AdGuard versions, the browser extension waits for https://github.com/AdguardTeam/tsurlfilter/issues/66
-! Part with "://*.*" before "$permissions=" is required, otherwise it will be marked as too wide rule
 ! AGLint temporarily disabled for this rule due to https://github.com/AdguardTeam/AGLint/issues/191
 ! TODO: Enable it back when AGLint is fixed.
-! TODO: remove $document modifier after versions with Corelibs 1.14.38+ will be released
-!#if (!ext_ublock)
 ! aglint-disable-next-line
-://*.*$document,permissions=join-ad-interest-group=()\,run-ad-auction=()\,browsing-topics=()
+*$permissions=join-ad-interest-group=()
```
Comment (AdamWr) [inline SpywareFilter/sections/general_extensions.txt:414]: In the past such rules were marked as too wide, has it been fixed?
>! Error: "The rule is too wide, add domain restriction or make the pattern more specific" in the rule: `"*.*$permissions=join-ad-interest-group=()\, run-ad-auction=()\, browsing-topics=()"`
`*.*$permissions=join-ad-interest-group=()\, run-ad-auction=()\, browsing-topics=()`

https://github.com/AdguardTeam/FiltersRegistry/blob/e529458be21800609358ce99ff1bcc8bb5b02305/filters/filter_3_Spyware/diff.txt

**Review by AdamWr — APPROVED** [review]

File: SpywareFilter/sections/general_extensions.txt (hunk 1 above)
Comment (Alex-302) [reply SpywareFilter/sections/general_extensions.txt:414]: Extension and app show them as applied)
Which blocker has this isssue?

Comment (AdamWr) [reply SpywareFilter/sections/general_extensions.txt:414]: As I recall, it wasn't a specified blocker but it was rejected in filters registry.

Comment (AdamWr) [reply SpywareFilter/sections/general_extensions.txt]: It was just a question, maybe it has been fixed already and `*$permissions=` can be used.

**Pull request merged**